migrations/Qi.sql

```diff
@@ -1,3 +1,143 @@
 -- 1 up
 
--- 1 down
\ No newline at end of file
+CREATE TABLE if not exists asset_types (
+  asset_type_id int NOT NULL primary key,
+  name varchar(255) DEFAULT NULL
+);
+
+CREATE TABLE if not exists assets (
+  tag varchar(7) NOT NULL DEFAULT '' primary key,
```

Comment:
Most of the other tables below define `primary key` after all of the columns.  This one defines it with the column.  I don't care which you do, but be consistent.